convert2rhel/unit_tests/checks_test.py

```diff
@@ -735,6 +735,17 @@ def test_bad_kernel_fingerprint(
     )
 
 
+def test_kernel_not_installed(caplog, monkeypatch):
+    run_subprocess_mocked = mock.Mock(spec=run_subprocess, return_value=(" ", 1))
+    monkeypatch.setattr(checks, "run_subprocess", run_subprocess_mocked)
+    _bad_kernel_package_signature("4.18.0-240.22.1.el8_3.x86_64") == True
```

Comment:
I believe you are missing a `assert` call here to actually validate that the return is actually `True`. 

```suggestion
    assert _bad_kernel_package_signature("4.18.0-240.22.1.el8_3.x86_64") == True
```

Reply:
you are right about that, thank you for catching that for me! I'll get that fixed and pushed.